src/swig/CMakeLists.txt

```diff
@@ -16,7 +16,40 @@ endif ()
 include_directories (${NLOPT_PRIVATE_INCLUDE_DIRS})
 set_source_files_properties (nlopt.i PROPERTIES CPLUSPLUS ON)
 
-if (NUMPY_FOUND AND Python_FOUND)
+if (NLOPT_PYTHON_SCIKIT_BUILD)
+  configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/python/CMakeLists.txt.in
+    ${CMAKE_CURRENT_BINARY_DIR}/python/CMakeLists.txt
+    @ONLY
+  )
+  configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/nlopt.i
+    ${CMAKE_CURRENT_BINARY_DIR}/python/nlopt.i
```

Comment:
Is there a good reason why these have to be copied?  Why can't they be used directly from the source dir?